ocean_lib/assets/test/test_utils.py

```diff
@@ -0,0 +1,98 @@
+#
```

Comment:
I don't think we gave much thought to that. Since these functions are helpers to deal with medium/big JSONs, I don't see removing an algo that doesn't exist as a security risk or blocker. It depends on how we want to deal with it:
- block the flow and spit out an error?
- fail silently?

Either way I'd make a separate issue for that, since this one is getting crowded (I think there's about 3 issues solved with one PR)

@trentmc what do you think?

Reply:
@trentmc Some guidance, please. We forgot about this when we discussed this PR.

Reply:
Separate issue is best, yes.

Reply:
will be addressed in https://github.com/oceanprotocol/ocean.py/issues/284